conf.d/openstack.yaml.example

```diff
@@ -0,0 +1,67 @@
+init_config:
+      # All directives prefixed with a '#' sign are optional and will default to sane values when omitted
+
+      # Where your identity server lives (please omit the trailing slash). Note that the server must support Identity API v3
+      keystone_server_url: "https://my-keystone-server.com:<port>"
```

Comment:
So with the recommendation of having 1 agent configured per physical host, we make the pretty important assumption that there will be exactly 1 hypervisor and 1 keystone server per physical host. It made sense to me to include the project-level config under `instances`, because one physical host can have many projects, but include the "global" configuration in `init_config` since they will be unique to the physical host, and won't change instance-by-instance. Let me know if you disagree